src/IECoreScene/LinkedScene.cpp

```diff
@@ -171,11 +172,12 @@ void LinkedScene::writeLink( const SceneInterface *scene )
 
 IECore::CompoundDataPtr LinkedScene::linkAttributeData( const SceneInterface *scene )
 {
-	std::string f = scene->fileName();
+	boost::filesystem::path nativePath( scene->fileName() );
+
 	InternedStringVectorDataPtr r = new InternedStringVectorData();
 	scene->path( r->writable() );
 	CompoundDataPtr d = new CompoundData();
-	d->writable()[g_fileName] = new StringData(f);
+	d->writable()[g_fileName] = new StringData( nativePath.generic_string() );
```

Comment:
Yes, `generic_string()` is the Boost method for ensuring paths always have forward slashes, regardless of platform. Prior to this change if a path had been supplied with back slashes it would have caused an error when trying to read the scene on Linux, ensuring any valid paths have forward slashes.

If anyone has used `LinkedScene` on Windows, their links would have been stored using back slashes, but now would be stored with forward slashes. I have it converting back to back slashes (on Windows) in the `readAttribute` method, so I don't think it shouldn't be causing any unexpected behavior.

Maybe the difference in storage of the link path warrants a breaking change?

Reply:
Ok, thanks for explaining. Since the problem could only have happened with .lscc files written on windows, I think we can safely call that an "experimental" case and ignore the breaking change